src/Microsoft.ML.Console/Microsoft.ML.Console.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>CORECLR</DefineConstants>
+	<IncludeInPackage>Microsoft.ML</IncludeInPackage>
+	<TargetFramework>netcoreapp2.0</TargetFramework>
+	<OutputType>Exe</OutputType>
+	<StartupObject>Microsoft.ML.Runtime.Tools.Console.Console</StartupObject>
```

Comment:
><StartupObject>Microsoft.ML.Runtime.Tools.Console.Console</StartupObject> [](start = 1, length = 73)

This would be the actual thing we'd do `dotnet` on, might be nice to name the build artifact something like `mml.dll` or something, just to keep usage here simple.

Reply:
You would do on Microsoft.ML.Runtime.Tools.Console.dll, do are you saying we should generate mml.dll instead?

---
In reply to: [192227732](https://github.com/dotnet/machinelearning/pull/248#discussion_r192227732) [](ancestors = 192227732)

Reply:
Yes, in things intended to be invoked from the command line I'd prefer something shorter. We're coming up with a replacement for `maml`, I might prefer either `mml` or `msml` or something, at least as preliminary names. Microsoft.ML.Runtime.Tools.Console.dll` doesn't exactly roll off the tongue. :P

---
In reply to: [192240765](https://github.com/dotnet/machinelearning/pull/248#discussion_r192240765) [](ancestors = 192240765,192227732)